src/app/features/book/bookSlice.js

```diff
@@ -0,0 +1,88 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const getBooksURL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/AvIE67KLnFDsdhR2CWwG/books';
+
+const addBooksURL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/AvIE67KLnFDsdhR2CWwG/books';
+
+const deleteBooksURL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/AvIE67KLnFDsdhR2CWwG/books/';
+
+const initialState = {
+  loading: false,
+  books: [],
+  error: '',
+};
+
+export const fetchBooks = createAsyncThunk('book/fetchBooks', async () => {
+  const { data } = await (axios.get(getBooksURL));
+  return data;
+});
+
+export const addBooks = createAsyncThunk('book/addBooks', async (obj) => {
+  const { data } = await (axios.post(addBooksURL, obj));
+  return data;
+});
+
+export const deleteBooks = createAsyncThunk('book/deleteBooks', async (id) => {
```

Comment:
- [ ] **Good job so far** However, It would be better to structure your files to follow the duck's pattern: 🙏 this will help our React-Redux project in improving organization, encapsulation, and maintainability of the codebase. 😀😀😀
  
![image](https://user-images.githubusercontent.com/102757126/222070091-6344b71c-35de-4ff1-a5e3-3b1ee4dca209.png)